tests/draft3/additionalItems.json

```diff
@@ -70,10 +85,10 @@
     },
     {
         "description": "additionalItems are allowed by default",
-        "schema": {"items": []},
+        "schema": {"items": [{"type": "integer"}]},
```

Comment:
Is this change intended?

Reply:
yes, see my comment just above. I didn't see why there should be a discrepancy in this file (but the draft3 spec is not even available anymore so maybe all draft3 tests should be removed?) :)

Reply:
Ah sorry, meaning this differed between the test as written for draft3 and later ones? Got it.